js/src/components/paid-ads/faqs-section.js

```diff
@@ -10,6 +10,17 @@ import { recordEvent } from '@woocommerce/tracks';
  */
 import Section from '.~/wcdl/section';
 
+/**
+ * Clicking on faq items to collapse or expand it in the Setup Ads page
+ *
+ * @event gla_setup_ads_faq
+ * @property {string} id (faq identifier)
+ * @property {string} action (`expand`|`collapse`)
```

Comment:
❓  Not sure if this parenthesis are necessary

Reply:
Good catch. I just copied the desciptions from the old `README.md` but I'll fix it here.